### DISCUSSION TITLE
Error response from daemon: Head "https://nvcr.io/v2/ohlfw0olaadg/ea-participants/cached/manifests/0.1.0": denied: {"errors": [{"code": "DENIED", "message": "Access Denied"}]}

### DISCUSSION THREAD

**Discussion by databloom** [NVIDIA/nv-ingest#168] [NV-Ingest uS]
docker compose up
[+] Running 8/11
 ✘ embedding Error             context canceled                                                                                                                                                               0.5s 
 ⠴ otel-collector [⠀⠀⠀] Pulling                                                                                                                                                                               0.5s 
 ! nv-ingest-ms-runtime Warning context canceled                                                                                                                                                              0.5s 
 ⠴ grafana [⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀] Pulling                                                                                                                                                                               0.5s 
 ⠴ prometheus [⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀] Pulling                                                                                                                                                                          0.5s 
 ✘ deplot Error                context canceled                                                                                                                                                               0.5s 
 ⠴ redis [⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀] Pulling                                                                                                                                                                            0.5s 
 ✘ cached Error                Head "https://nvcr.io/v2/ohlfw0olaadg/ea-participants/cached/manifests/0.1.0": denied: {"errors": [{"code": "DENIED", "message": "Access Denied"}]}                            0.5s 
 ✘ yolox Error                 context canceled                                                                                                                                                               0.5s 
 ✘ paddle Error                context canceled                                                                                                                                                               0.5s 
 ⠴ zipkin [⠀⠀⣿⠀⠀⠀⠀⠀⠀] Pulling                                                                                                                                                                                 0.5s 
Error response from daemon: Head "https://nvcr.io/v2/ohlfw0olaadg/ea-participants/cached/manifests/0.1.0": denied: {"errors": [{"code": "DENIED", "message": "Access Denied"}]}

No problem pulling ngc containers with my .env settings, (docker login is working) just the nv-ingest specific one in the docker-compose.yaml

**Comment by jarmak-nv** [comment]
Hi! Very sorry about the delay, but NV-Ingest NIMs are in EA currently and require you to be in the EA program.

Please apply through the microservices page here: https://developer.nvidia.com/nemo-microservices
